lib/jekyll/filters.rb

```diff
@@ -363,5 +384,20 @@ def as_liquid(item)
         end
       end
     end
+
+    # Parse a string to a Liquid Condition
+    def parse_condition(exp)
+      parser = Liquid::Parser.new(exp)
+      a = parser.expression
+      if op = parser.consume?(:comparison)
+        b = parser.expression
+        condition = Liquid::Condition.new(a, op, b)
```

Comment:
Hmm, is that in our Rubocop setup? I don't like either of those personally – I find them both harder to read than the current version. Indentation in the first example doesn't help me know that the `if` statement affects `var` and the second one is noisy. I think with better variable names (not just `a`, `b`, and `op`), we can make this code more readable.